### ISSUE TITLE
Unable to retrieve available updates: unknown version 4.1.0-rc.0.

### ISSUE THREAD

**Issue by davivcgarcia** [openshift/installer#1677]
# Version

```console
$ openshift-install version
openshift-install v4.1.0-201904211700-dirty
built from commit f3b726cc151f5a3d66bc7e23e81b3013f1347a7e
release image quay.io/openshift-release-dev/ocp-release@sha256:345ec9351ecc1d78c16cf0853fe0ef2d9f48dd493da5fdffc18fa18f45707867
```

# Platform (aws|libvirt|openstack):

AWS

# What happened?

After a default install using `openshift-install create cluster` using the [4.1.0-rc0 release ](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-install-linux-4.1.0-rc.0.tar.gz), the cluster is not able to retrieve updates with the error `Could not retrieve updates. Unable to retrieve available updates: unknown version 4.1.0-rc.0.`. 

# What you expected to happen?

Cluster being able to track updates.

# How to reproduce it (as minimally and precisely as possible)?

1. `openshift-install create cluster`
2. Check for updates

# Anything else we need to know?

```console
$ oc --kubeconfig=auth/kubeconfig describe clusterversion -n cluster-api
Name:         version
Namespace:    
Labels:       <none>
Annotations:  <none>
API Version:  config.openshift.io/v1
Kind:         ClusterVersion
Metadata:
  Creation Timestamp:  2019-04-25T13:47:19Z
  Generation:          1
  Resource Version:    12956
  Self Link:           /apis/config.openshift.io/v1/clusterversions/version
  UID:                 a554a389-6760-11e9-978e-02ec3fb9726e
Spec:
  Channel:     stable-4.0
  Cluster ID:  d318a0ce-2f16-44db-a787-1a4bc500c784
  Upstream:    https://api.openshift.com/api/upgrades_info/v1/graph
Status:
  Available Updates:  <nil>
  Conditions:
    Last Transition Time:  2019-04-25T13:57:20Z
    Message:               Done applying 4.1.0-rc.0
    Status:                True
    Type:                  Available
    Last Transition Time:  2019-04-25T13:48:08Z
    Status:                False
    Type:                  Failing
    Last Transition Time:  2019-04-25T13:57:20Z
    Message:               Cluster version is 4.1.0-rc.0
    Status:                False
    Type:                  Progressing
    Last Transition Time:  2019-04-25T13:48:08Z
    Message:               Unable to retrieve available updates: unknown version 4.1.0-rc.0
    Reason:                RemoteFailed
    Status:                False
    Type:                  RetrievedUpdates
  Desired:
    Image:    quay.io/openshift-release-dev/ocp-release@sha256:345ec9351ecc1d78c16cf0853fe0ef2d9f48dd493da5fdffc18fa18f45707867
    Version:  4.1.0-rc.0
  History:
    Completion Time:    2019-04-25T13:57:20Z
    Image:              quay.io/openshift-release-dev/ocp-release@sha256:345ec9351ecc1d78c16cf0853fe0ef2d9f48dd493da5fdffc18fa18f45707867
    Started Time:       2019-04-25T13:48:08Z
    State:              Completed
    Version:            4.1.0-rc.0
  Observed Generation:  1
  Version Hash:         jHX1796OCic=
Events:                 <none>
```

**Comment by davivcgarcia** [comment]
I just noticed that `ClusterVersion` is tracking the `stable-4.0` channel by default. I changed to `stable-4.1` but I'm not sure how long it takes to try again (the error message is still from the last try).

```console
$ oc --kubeconfig=auth/kubeconfig describe clusterversion -n cluster-api
Name:         version
Namespace:    
Labels:       <none>
Annotations:  <none>
API Version:  config.openshift.io/v1
Kind:         ClusterVersion
Metadata:
  Creation Timestamp:  2019-04-25T13:47:19Z
  Generation:          2
  Resource Version:    48980
  Self Link:           /apis/config.openshift.io/v1/clusterversions/version
  UID:                 a554a389-6760-11e9-978e-02ec3fb9726e
Spec:
  Channel:     stable-4.1
  Cluster ID:  d318a0ce-2f16-44db-a787-1a4bc500c784
  Upstream:    https://api.openshift.com/api/upgrades_info/v1/graph
Status:
  Available Updates:  <nil>
  Conditions:
    Last Transition Time:  2019-04-25T13:57:20Z
    Message:               Done applying 4.1.0-rc.0
    Status:                True
    Type:                  Available
    Last Transition Time:  2019-04-25T13:48:08Z
    Status:                False
    Type:                  Failing
    Last Transition Time:  2019-04-25T13:57:20Z
    Message:               Cluster version is 4.1.0-rc.0
    Status:                False
    Type:                  Progressing
    Last Transition Time:  2019-04-25T13:48:08Z
    Message:               Unable to retrieve available updates: unknown version 4.1.0-rc.0
    Reason:                RemoteFailed
    Status:                False
    Type:                  RetrievedUpdates
  Desired:
    Image:    quay.io/openshift-release-dev/ocp-release@sha256:345ec9351ecc1d78c16cf0853fe0ef2d9f48dd493da5fdffc18fa18f45707867
    Version:  4.1.0-rc.0
  History:
    Completion Time:    2019-04-25T13:57:20Z
    Image:              quay.io/openshift-release-dev/ocp-release@sha256:345ec9351ecc1d78c16cf0853fe0ef2d9f48dd493da5fdffc18fa18f45707867
    Started Time:       2019-04-25T13:48:08Z
    State:              Completed
    Version:            4.1.0-rc.0
  Observed Generation:  1
  Version Hash:         jHX1796OCic=
Events:                 <none>
```

**Comment by wking** [comment]
We don't expect 4.1.0-rc.0 in the `stable-4.0` channel.  Closing in favor of #1599, although the new release isn't in `stable-4.1` yet either:

```console
$ curl -H 'Accept: application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.1'
{"nodes":[],"edges":[]}
```

so we can't land that PR quite yet.  Hopefully soon.

/close
